app/ui/browser/views/navbar/location.jsx

```diff
@@ -163,6 +163,17 @@ export class Location extends Component {
     return 'glyph-bookmark-hollow-16.svg';
   }
 
+  getVisibleCompletionsForURL() {
```

Comment:
We use the word location in other places, like userTypedLocation. Should we be consistent?